### PR TITLE
config: Alphabetize lists for easier maintenance

### PR DESCRIPTION
**Pull request by lblackstone**
No functional change; just re-ordered variables.

### CODE REVIEW

**Comment by s-urbaniak** [comment]
ok to test

**Comment by lblackstone** [comment]
@s-urbaniak Had to rebase. Can we get this merged today, as this type of PR tends to require a lot of manual rebasing? Thanks!

File: config.tf [hunk 1]
```diff
@@ -68,12 +68,12 @@ variable "tectonic_versions" {
   type        = "map"
 
   default = {
+    alertmanager    = "v0.7.1"
     container_linux = "1353.8.0"
     etcd            = "3.1.8"
-    prometheus      = "v1.7.1"
-    alertmanager    = "v0.7.1"
+    kubernetes      = "1.6.7+tectonic.1"
```
Comment (lblackstone) [inline config.tf]: @s-urbaniak Need to double-check this version. Looks like it was reverted to `1.6.6+tectonic.1` in https://github.com/coreos/tectonic-installer/pull/1154, but that may have been a mistake.

Comment (s-urbaniak) [inline config.tf]: @lblackstone very good catch! that was indeed mistakenly reverted, I will follow up with a PR.

File: config.tf [hunk 2]
```diff
@@ -68,12 +68,12 @@ variable "tectonic_versions" {
...
     monitoring      = "1.3.0"
-    kubernetes      = "1.6.6+tectonic.1"
+    prometheus      = "v1.7.1"
     tectonic        = "1.6.6-tectonic.1"
```
Comment (s-urbaniak) [inline config.tf]: for this PR we also have to bump to `1.6.7-tectonic.1`

**Comment by lblackstone** [comment]
@s-urbaniak RFR again

**Comment by squat** [comment]
@lblackstone sorry for the delay, @s-urbaniak is OOF currently.

**Pull request merged**